### ISSUE TITLE
SGB format is incorrectly described in Help document

### ISSUE THREAD

**Issue by HeyItsLollie** [Rangi42/tilemap-studio#71]
In Tilemap Studio's Help document, SGB format is described:
> SGB border: Bytes come in pairs; a tile ID and an attribute byte. The only supported attribute bits are ***a choice of four palettes (each with four colors)*** and X/Y flip.

The palette information is true for the Game Boy portion of the screen, but incorrect for the actual border. From GBDev's Pan Docs:
https://gbdev.io/pandocs/SGB_Color_Palettes.html
> The SGB/SNES provides 8 palettes of 16 colors each, each color may be defined out of a selection of 32768 colors (15 bit). Palettes 0-3 are used to colorize the gamescreen, only the first four colors of each of these palettes are used. ***Palettes 4-7 are used for the SGB Border, all 16 colors of each of these palettes may be used.***

**Comment by HeyItsLollie** [comment]
After attempting another test today (while using Tools > Image to Tools), I've realized that Tilemap Studio itself incorrectly assumes that Super Game Boy borders only use four colors per tile. They're technically the same as SNES tiles, 4bpp.

**Comment by Rangi42** [comment]
Yeah, the "SGB border" format was intended for the limited Pokémon Gen 1 and 2 borders. Just using the "SNES tiles + attributes" format for fully-general SGB borders should work.

I'll update the docs to clarify this, since there's no use in having the two formats be identical.
